### PR TITLE
Allow multiple Web3Signer Validator Clients with same database

### PR DESCRIPTION
**Pull request by dominikkaminski**
Hey all,

we want to run multiple validator clients with the same web3signer and its database so we like to slice the keys.

I would suggest to add an start index to this chart to allow multiple deployments.

### CODE REVIEW

**Comment by unxnn** [comment]
> Hey all,
> 
> we want to run multiple validator clients with the same web3signer and its database so we like to slice the keys.
> 
> I would suggest to add an start index to this chart to allow multiple deployments.

Hey @dominikkaminski, definitely a great idea! Added to our Todo list

File: charts/web3signer-validators/templates/statefulset.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{- if .Values.enabled }}
 {{- $root := . -}}
 {{- $counter := 0 }}
-{{- range (untilStep 0 (int .Values.validatorsCount) 1) }}
+{{- range (untilStep (int .Values.validatorsKeyIndex) (add .Values.validatorsKeyIndex .Values.validatorsCount) 1) }}
```
Comment (unxnn) [inline charts/web3signer-validators/templates/statefulset.yaml:4]: Hey @dominikkaminski, definitely like this, thanks for the PR. Are you sure this is correct? Getting this error:

```
Error: template: web3signer-validators/templates/statefulset.yaml:4:93: executing "web3signer-validators/templates/statefulset.yaml" at <.Values.validatorsCount>: wrong type for value; expected int; got int64
helm.go:84: [debug] template: web3signer-validators/templates/statefulset.yaml:4:93: executing "web3signer-validators/templates/statefulset.yaml" at <.Values.validatorsCount>: wrong type for value; expected int; got int64
```

Using these values:

```
validatorsCount: 5
validatorsKeyIndex: 5
```

**Review by unxnn — APPROVED** [review]

File: charts/web3signer-validators/templates/statefulset.yaml (hunk 1 above)
Comment (dominikkaminski) [reply charts/web3signer-validators/templates/statefulset.yaml]: Hey @unxnn this time you were faster in testing as I was :pensive: 

Now is tested and the issue is fixed (i didnt expect that untilStep could not handle the return of int64 from add)

**Comment by unxnn** [comment]
Thank you @dominikkaminski looks ok now 👍

**Pull request merged**